apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -220,6 +220,23 @@ def _convert_phrases(raw_phrases) -> List[Phrase]:
     return list(map(_convert_phrase, raw_phrases))
 
 
+def _convert_two_way_dictionary(data) -> List[tuple]:
+    """
+    Handles loading the twoway-dictionary form the YAML format
+    """
+    dictionary: List[tuple] = []
+    # return dictionary
+    if "Twoway-dictionary" not in data:
+        return dictionary
+    raw_two_way_dictionary = data["Twoway-dictionary"]
```

Comment:
```suggestion
    raw_two_way_dictionary = data["Two-way dictionary"]
```

then the only remaining thing I'd suggest is using this spelling